docs/UserGuide.adoc

```diff
@@ -130,17 +243,238 @@ Examples:
+
+****
+* Deletes the policy at the specified `INDEX`.
+* The index refers to the index number shown in the displayed policy list.
+* The index *must be a positive integer* 1, 2, 3, ...
+****
+
+Examples:
+
+* `listpolicies` +
+`delete 2 +
+Deletes the 2nd policy in the list of policies.
+* `find senior` +
+`delete 1` +
+Deletes the 1st policy in the results of the `find` command.
+
+=== Deleting policies from a person : `unassignpolicy`
+
+Removes a policy from the person at the specified index. +
+Format: `unassignpolicy INDEX pol/POLICY [MORE_POLICIES]`
+
+****
+* Removes the policies to the person at the specified `INDEX`.
+* The index refers to the index number shown in the displayed person list.
+* The index *must be a positive integer* 1, 2, 3, ...
+****
+
+Examples:
+
+* `listpeople` +
+`unassignpolicy 2 pol/Senior Care` +
+Removes the `Senior Care` policy from the 2nd person in the list of people.
+* `find Betsy` +
+`unassignpolicy 1 pol/Health Insurance` +
+Removes the `Health Insurance` policy from the 1st person in the results of the `find` command.
+
+=== Deleting tags from a person : `deletetag`
+
+Deletes a tag from the person at the specified index. +
+Format: `deletetag INDEX t/TAG [MORE_TAGS]`
+
+****
+* Deletes the tags from the person at the specified `INDEX`.
+* The index refers to the index number shown in the displayed person list.
+* The index *must be a positive integer* 1, 2, 3, ...
+* Any number of tags can be deleted.
+****
+
+Examples:
+
+* `listpeople` +
+`deletetag 2 t/high_priority` +
+Deletes a `high_priority` tag from the 2nd person in the list of people.
+* `find Betsy` +
+`deletetag 1 t/high_risk` +
+Deletes a `high_risk` tag from the 1st person in the results of the `find` command.
+
+=== Deleting tags from a policy : `deletepolicytag`
+
+Deletes a tag from the policy at the specified index. +
+Format: `deletepolicytag INDEX t/TAG [MORE_TAGS]`
+
+****
+* Deletes the tags from the policy at the specified `INDEX`.
+* The index refers to the index number shown in the displayed policies list.
+* The index *must be a positive integer* 1, 2, 3, ...
+* Any number of tags can be deleted.
+****
+
+Examples:
+
+* `listpolicies` +
+`deletepolicytag 2 t/high_priority` +
+Deletes a `high_priority` tag from the 2nd policy in the list of policies.
+* `findpolicy Senior` +
+`deletepolicytag 1 t/high_risk` +
+Deletes a `high_risk` tag from the 1st policy in the results of the `find` command.
+
 // end::delete[]
 === Clearing all entries : `clear`
 
 Clears all entries from the address book. +
 Format: `clear`
 
+=== Incorrect NRIC Alert
+
+Returns an error message to inform the user of an invalid NRIC. The contact number and/or email address is returned
+if present.
+
+****
+* The details of the input person will not be added until all information is valid.
+****
+
+Example:
+
+`add n/John Doe ic/Q9999999J p/98765432 e/johnd@example.com a/John street, block 123, #01-01 b/12.09.1980 pol/SeniorCare t/high-priority` +
```

Comment:
Maybe we can put this as a block of code instead? standardise the look with the expected output

Reply:
oh i thought we should distinguish between user input and the output and usually user input is only one line so can leave it like that? what do you think?

Reply:
Okay sure, this makes sense.